docs/technical_documentation/concepts/ralph.rst

```diff
@@ -12,7 +12,7 @@ Although Ralph has usages such as:
 - Convert tracking logs to xAPI statements.
 - Fetch xAPI statements from a configured backend.
 - Validate xAPI statements.
-- Store events to different `backends <https://openfun.github.io/ralph/backends/>`_.
+- Store events to different backends.
```

Comment:
here's the updated link to that page:

https://openfun.github.io/ralph/latest/features/backends/